README.md

```diff
@@ -63,6 +63,21 @@ kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockerc
 kubectl create configmap snyk-monitor-certs -n snyk-monitor --from-file=<path_to_certs_folder>
 ```
 
+5. If you are using an insecure registry or your registry is using unqualified images, you can provide `registries.conf` file. It is a system-wide configuration file for container image registries. The file format is TOML. For more information on how to format and examples visit: https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md 
```

Comment:
Some minor edits to the prose:

5. If you are using an insecure registry or your registry is using unqualified images, you can provide a `registries.conf` file. See [the documentation](https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md ) for information on the format and examples.